src/mocking-server.js

```diff
@@ -5,7 +5,7 @@ export type Request = {
     method: string,
     urlPath: string,
     urlParams: ParamsBag,
-    formFields: ParamsBag,
+    formFields: ParamsBag | void,
```

Comment:
can't we default to {} to not change the interface?

Reply:
sure, changed